### PR TITLE
New Block Attestation Orderer GRPC service

### PR DESCRIPTION
**Pull request by Param-S**
#### Type of change
- New feature

#### Description
New RPC service for block attestation is added. It will be used by BFT block replication

#### Related issues
#3413 

Signed-off-by: Parameswaran Selvam <parselva@in.ibm.com>

### CODE REVIEW

File: orderer/common/server/main.go [hunk 1]
```diff
@@ -278,6 +278,15 @@ func Main() {
 		conf.General.Authentication.NoExpirationChecks,
 	)
 
+	attestationService := NewAttestationService(
```
Comment (yacovm) [inline orderer/common/server/main.go]: Let's put it optional for now, and disable it by default?

Comment (yacovm) [reply orderer/common/server/main.go]: Or we can not attach it for now

Comment (Param-S) [reply orderer/common/server/main.go]: Updated. The service is now not enabled.  

But, still the dead/unused code still be part of the other future releases of 2.x.  can we move this BFT changes into separate branch.

**Comment by yacovm** [comment]
```
The following files contain gofumpt errors
orderer/common/server/attestationserver_test.go
The gofumpt command 'gofumpt -l -s -w' must be run for these files

```

File: orderer/common/server/main.go (hunk 1 above)
Comment (yacovm) [reply orderer/common/server/main.go]: if it's not imported from the main, it is not part of the binary. I don't think we need to put it in a separate branch.

**Review by yacovm — APPROVED** [review]

**Pull request merged**